code/modules/mob/living/simple_animal/hostile/mining_mobs/goliath.dm

```diff
@@ -99,6 +100,38 @@
 	loot = list()
 	stat_attack = UNCONSCIOUS
 	robust_searching = 1
+	food_type = list(/obj/item/reagent_containers/food/snacks/customizable/salad/ashsalad, /obj/item/reagent_containers/food/snacks/customizable/soup/ashsoup)//use lavaland plants to feed the lavaland monster
```

Comment:
so it won't eat raw lavaland flora?

Reply:
no

Reply:
(Warning, lore, ecosystem theory, and logic ahead)
Omnivorous goliaths?
That would kinda make sense in this alien ecosystem where there's way more carnivores than herbivores (closer to **no** herbivores) because goliaths would be gobbling up cacti until they find more nutritious food (say, humans) and they'd make the base food for all the other creatures, up until apex megafauna. 
The not eating raw plants part doesn't make much sense though. A goliath's stomach would have evolved to be able to withstand a freshly scraped off mushroom leaf. Why would it reject raw plants, that are full with nutrition?